.github/actions/get_repo_name/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Get Repo Name'
+description: 'Gets the repository name that was added to the open-repositories.txt file.'
+outputs:
+  repo-name:
+    value: ${{ steps.get-repo-name.outputs.new_repo }}
+    description: "The name of the repository that was added."
+  new_repo_added:
+    value: ${{ steps.get-repo-name.outputs.new_repo_added }}
+    description: "Boolean if a new repo was added."
+
+runs:
+  using: 'composite'
+  steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get repo name
+        id: get-repo-name
+        run: |
+          NEW_REPO=$(git diff origin/master...HEAD open-repositories.txt | grep "^+[a-zA-Z]" | sed 's/+//' || echo "")
+          if [ "$NEW_REPO" = "" ]; then
```

Comment:
You should try `-n` instead - [bash reference](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html#Bash-Conditional-Expressions)